enso/sample/__init__.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
I'd beef up the README to explain what has to be implemented in samplers and what's available in them.

Doc strings for the sampler methods would be nice as well, especially for something like distances where I won't always want to use the base method (but I will frequently want to use it) adding information both to the README and the file itself on grabbing the bits you want while over-riding the bits you don't is helpful.